### PR TITLE
Updated the footer to make it sticky.

### PR DESCRIPTION
**Pull request by mbockus**
Replaced the placeholder footer with a sticky footer and moved the About and Terms nav from the nav bar to the footer. This PR is to resolve issue #72.

### CODE REVIEW

File: school-finder-frontend/app/index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html ng-app="sfApp">
 <head>
 	<title>School Finder</title>
+    <link href="/favicon.ico" type="image/x-icon" rel="icon" />
```
Comment (joekarl) [inline school-finder-frontend/app/index.html]: This won't resolve correctly unless the app is run off of a root path as it's an absolute path. 
If we switch this to be `href="favicon.ico"` that should do the trick.

Also indentation.

**Comment by joekarl** [comment]
Just the one comment, but looks good otherwise. Thanks!

**Comment by mbockus** [comment]
Ah, good catch on the indentation and absolute reference. The PR has been updated.

**Comment by joekarl** [comment]
Thanks @mbockus!

**Pull request merged**